examples/misc_camera_traveling.html

```diff
@@ -0,0 +1,202 @@
+    		    <li>Ctrl + Left-Click: camera rotation (orbit)</li>
+    		    <li>Mouse Wheel: zoom in/out</li>
+    		    <li>S: save current camera position</li>
+    		    <li>T: travel through each saved camera positions</li>
+    		    <li>T while traveling: interrupt travel</li>
+    		    <li>C: clear all saved camera positions</li>
+	        </ul>
+	    </div>
+        <div id="viewerDiv"></div>
+
+        <script src="js/GUI/GuiTools.js"></script>
+        <script src="../dist/itowns.js"></script>
+        <script src="js/GUI/LoadingScreen.js"></script>
+        <script src="../dist/debug.js"></script>
+        <script type="text/javascript">
+            // # Simple Globe viewer
+            /* global itowns */
+
+            // `viewerDiv` will contain iTowns rendering area (`<canvas>`) :
+            let viewerDiv = document.getElementById('viewerDiv');
+
+            // Define initial camera position :
+            let placement = {
+                coord: new itowns.Coordinates('EPSG:4326', 2.351323, 48.856712),
+                range: 50000
+            };
+
+            // Instantiate iTowns GlobeView :
+            let view = new itowns.GlobeView(viewerDiv, placement);
+
+            // Sets a loading screen :
+            setupLoadingScreen(viewerDiv, view);
+
+            // Add one imagery layer to the scene and the miniView.
+            // This layer is defined in a json file but it could be defined as a plain js
+            // object. See Layer* for more info.
+            itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(function _(config) {
+                config.source = new itowns.WMTSSource(config.source);
+                const orthoLayer = new itowns.ColorLayer('Ortho', config);
+                view.addLayer(orthoLayer);
+            });
+
+            // Add two elevation layers.
+            // These will deform iTowns globe geometry to represent terrain elevation.
+            function addElevationLayerFromConfig(config) {
+                config.source = new itowns.WMTSSource(config.source);
+                let layer = new itowns.ElevationLayer(config.id, config);
+                view.addLayer(layer);
+            }
+            itowns.Fetcher.json('./layers/JSONLayers/WORLD_DTM.json').then(addElevationLayerFromConfig);
+            itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addElevationLayerFromConfig);
+
+            // Add a menu :
+            let menuGlobe = new GuiTools('menuDiv', view);
+            // Add an option in the menu to display picked position as 3D axes :
+            const displayPositions = menuGlobe.addGUI('PickedPositions', false, function (v) {
+                if (v) {
+                    view.scene.add(pickedPointsGroup);
+                    view.notifyChange();
+                } else {
+                    view.scene.remove(pickedPointsGroup);
+                    view.notifyChange();
+                }
```

Comment:
use visible property to hide/display this object
 
`view.notifyChange();` could be written only once

Reply:
This part has been changed.